var/spack/repos/builtin.mock/packages/gcc/package.py

```diff
@@ -11,9 +12,10 @@ class Gcc(CompilerPackage, Package):
     homepage = "http://www.example.com"
     url = "http://www.example.com/gcc-1.0.tar.gz"
 
-    version("1.0", md5="0123456789abcdef0123456789abcdef")
-    version("2.0", md5="abcdef0123456789abcdef0123456789")
+    version("14.0", md5="abcdef0123456789abcdef0123456789")
     version("3.0", md5="def0123456789abcdef0123456789abc")
+    version("2.0", md5="abcdef0123456789abcdef0123456789")
+    version("1.0", md5="0123456789abcdef0123456789abcdef")
 
     variant(
```

Comment:
Should compiler plugin support be a variant for GCC (and also LLVM-based compilers)? In each case it is possible for the compiler (especially if external) to be installed without the headers/libs needed to build plugins. See without too much loss of generality #42497 for a relevant use case.

Reply:
Is this a general question on `gcc`? Not sure what you mean here, since you commented on a mock package used for unit tests, and linked an issue talking about `python+tkinter` :thinking:

Reply:
Sorry, I meant to put this on `var/spack/repos/builtin/packages/gcc/package.py`, and have updated the PR number to what should be correct (Score-P getting `+gcc-plugin` as a variant; life there becomes simpler if we can depend on `^gcc+plugin-support` or similar). So yes, a general `gcc` question.